resources/ory/charts/hydra/files/watch.sh

```diff
@@ -0,0 +1,13 @@
+set -eu
```

Comment:
I have removed this shell script with the sidecar container in this PR 
- https://github.com/kyma-project/kyma/pull/13705
We are controlling this logic in ory-reconciler